package.json

```diff
@@ -162,9 +162,10 @@
     "test:assets": "yarn prettier --list-different",
     "test:auth": "yarn bundle:test && karma start karma.conf.react.js",
     "test:scripts": "eslint -c .eslintrc.json --ignore-path .gitignore --ignore-path .eslintignore app/script/ test/ bin/ grunt/ *.{js,jsx}",
-    "translate": "yarn translate:buildpot && yarn translate:download && yarn translate:postprocess && yarn translate:extractpo && yarn fix:assets",
+    "translate": "yarn translate:upload && yarn translate:download",
     "translate:buildpot": "rip json2pot \"temp/i18n/**/*.json\" -o \"temp/i18n/webapp.pot\"",
-    "translate:download": "node ./bin/translations_download.js",
+    "translate:download": "node ./bin/translations_download.js && yarn translate:postprocess && yarn translate:extractpo && yarn fix:assets",
+    "translate:upload": "yarn translate:buildpot && node ./bin/translations_upload.js",
```

Comment:
💯 Thx for making seperate upload and download available again. Makes the process much faster for me.